autolab/Autolab
# Fix bug where course gets created even if there are errors

## Description

- Added 'required' fields to HTML form fields for display name and instructor emails
- Destroys course data if instructor email is invalid

## Motivation and Context


Resolves #1714 

## How Has This Been Tested?



To test changes: 
- Login as admin account
- Go to manage course -> create new course
- Case 1:
    - Enter course name
    - Leave other fields blank
    - Click "create" (if you want you can go to home and see that the course has not been created)
    - Enter valid email
    - Click "create"
- Case 2:
    - Enter course name
    - Enter invalid email
    - Click "create" (if you want you can go to home and see that the course has not been created)
    - Enter valid email
    - Click "create" 

**BEFORE**

https://user-images.githubusercontent.com/45646252/220240285-20b8050d-fb44-4cc2-ad3f-9c739269f636.mp4

**AFTER**

https://user-images.githubusercontent.com/45646252/220240691-10b06ff5-aaaf-4511-9a25-2c4d29da783e.mp4



## Types of changes



## Checklist:
